### PR TITLE
Update Postgres Exporter to v0.10.0

### PR DESCRIPTION
**Pull request by rgeyer**
#### PR Description

Bumps the version of the postgres exporter to resolve some performance issues for existing users.

#### PR Checklist

- [x] CHANGELOG updated 
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,7 +9,7 @@ require (
 	github.com/davidmparrott/kafka_exporter/v2 v2.0.1
 	github.com/drone/envsubst v1.0.2
 	github.com/fatih/structs v1.1.0
-	github.com/go-kit/kit v0.10.0
+	github.com/go-kit/kit v0.11.0
```
Comment (rgeyer) [inline go.mod:12]: There are a few drive-by updates in go.mod and go.sum. I'll confess to still being a bit of a n00b WRT go modules, so I *think* these are okay, but I am not 100% sure.

The agent still builds. and tests still pass, so I think we're good.

Part of the extensive nature of the change here is that the postgres exporter repo changed from `github.com/wrouesnel/postgres_exporter` to `github.com/prometheus-community/postgres_exporter`.

Due to my go mod inexperience, I did a `go mod tidy` which... May be wrong.

File: pkg/integrations/postgres_exporter/postgres_exporter.go [hunk 2]
```diff
@@ -22,6 +22,7 @@ type Config struct {
 	DisableSettingsMetrics bool     `yaml:"disable_settings_metrics,omitempty"`
 	AutodiscoverDatabases  bool     `yaml:"autodiscover_databases,omitempty"`
 	ExcludeDatabases       []string `yaml:"exclude_databases,omitempty"`
+	IncludeDatabases       []string `yaml:"include_databases,omitempty"`
```
Comment (rgeyer) [inline pkg/integrations/postgres_exporter/postgres_exporter.go:25]: I deliberately did *not* add the "metricPrefix" option which is new in the latest version of the exporter. My thinking is that we want to be opinionated about metric names/namespaces in the agent.

**Review by rgeyer — COMMENTED** [review]
Couple things...

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Main (unreleased)
 
+- [ENHANCEMENT] Update postgres_exporter to a [branch of](https://github.com/grafana/postgres_exporter/tree/exporter-package-v0.10.0) v0.10.0
```
Comment (rfratto) [inline CHANGELOG.md:3]: Can you move this below the other enhancements? We try to keep the CHANGELOG sorted first by FEATURE > ENHANCEMENT > BUGFIX > CHANGE > DEPRECATION, and then by date per category.

**Review by mattdurham — APPROVED** [review]

**Pull request merged**